incompatibility/reporter_test.go

```diff
@@ -66,57 +66,313 @@ func makePathsObject(pathsName string, operationType ...InvalidOperationType) *o
 			pathItem.Trace = &openapiv3.Operation{}
 		}
 	}
+	pathItem.Summary = "Sudo Summary"
+	pathItem.Description = "Sudo Description"
 	return &openapiv3.Paths{Path: []*openapiv3.NamedPathItem{{Name: pathsName, Value: pathItem}}}
 }
 
-// Simple test for in-progress incompatibility chain coverage
-func TestReporterCoverage(t *testing.T) {
+func testIncompatibilityReports(t *testing.T, formattedError string, want, got *IncompatibilityReport) {
```

Comment:
This method might fall into: go/too-dry

I think it's right on the edge, but that is a great right up that I just wanted to post.